BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/GiniColors.swift

```diff
@@ -155,3 +156,106 @@ extension UIColor {
         }
     }
 }
+
+extension UIColor {
+    // MARK: Development scheme
+    // TODO: Need to update scheme when designer will finish it
+    public struct GiniColorScheme {
+        struct Bg {
```

Comment:
[suggestion] We should avoid abbreviations in naming something in our code.
Include all the words needed to avoid ambiguity for a person reading code where the name is used.
I don't understand if this should be background or something else since is containing also border which is not a background.